handsontable/src/plugins/hiddenRows/__tests__/plugins/mergeCells.spec.js

```diff
@@ -1804,26 +1806,29 @@ describe('HiddenRows', () => {
         colHeaders: true,
         contextMenu: true,
         hiddenRows: {
-          rows: [1],
+          rows: [0],
           indicators: true
         },
         mergeCells: [{ row: 1, col: 1, rowspan: 3, colspan: 1 }]
       });
 
-      const mergeArea = spec().$container.find('tr:eq(2) td:eq(1)');
+      const mergeArea = spec().$container.find('tr:eq(1) td:eq(1)');
 
       // Selected 2 ranges containing together merged area.
-      selectRows(1);
-      keyDown('ctrl');
-      selectRows(3);
-      keyDown('ctrl');
-      selectRows(2);
+      simulateClick(getCell(1, -1));
+
+      keyDown('control');
+
+      simulateClick(getCell(3, -1));
+      simulateClick(getCell(2, -1));
```

Comment:
```suggestion
      listen();
      selectRows(1);
      keyDown('control');
      selectRows(3);
      keyDown('control');
      selectRows(2);
```
The same [as here](https://github.com/handsontable/handsontable/pull/8955/files#r810891740). I would revert all the changes.

Reply:
As in https://github.com/handsontable/handsontable/pull/8955#discussion_r811899513.